inttest/backup/backup_test.go

```diff
@@ -155,6 +161,25 @@ func (s *BackupSuite) makeSnapshot(kc *kubernetes.Clientset) snapshot {
 	}
 }
 
+func (s *BackupSuite) VerifyFileSystemRestore() error {
+	ssh, err := s.SSH(s.ControllerNode(0))
+	if err != nil {
+		return err
+	}
+	defer ssh.Disconnect()
+
+	// Checking for containerd should be enough given https://github.com/k0sproject/k0s/issues/2420
+	// containerd may take a bit to start so we want to retry a few times
+	checkPID := func() bool {
+		_, err = ssh.ExecWithOutput(s.Context(), "/bin/pidof /var/lib/k0s/bin/containerd")
+		return err == nil
+	}
+
+	assert.Eventuallyf(s.T(), checkPID, 18*time.Second, 10*time.Second,
```

Comment:
1. Shouldn't this be 180 seconds?
2. Suites support assertions out of the box

```suggestion
	s.Eventuallyf(checkPID, 180*time.Second, 10*time.Second,
```

Reply:
D'oh! yes it should. I broke the tests on purpose to test that they actually worked and I forgot about this one. Sorry.